include/deal.II/base/function_lib.h

```diff
@@ -1254,6 +1254,81 @@ namespace Functions
      */
     const Table<dim,double>                     data_values;
   };
+
+
+  /**
+   * A class that represents a function object for a polynomial. A polynomial
+   * is composed by multiple monomials,i-e. in 1-d they have the form
+   * $x^\alpha + x^\beta + ... + x^\epsilon$, in 2-d the form
+   * $x_1^{\alpha_1}x_2^{\alpha_2} + x_1^{\beta_1}x_2^{\beta_2} + ... +
+   * x_1^{\epsilon_1}x_2^{\epsilon_2}$, and in 3-d
+   * $x_1^{\alpha_1}x_2^{\alpha_2}x_3^{\alpha_3} + x_1^{\beta_1}x_2^{\beta_2}x_3^{\beta_3}
+   * + ... + x_1^{\epsilon_1}x_2^{\epsilon_2}x_3^{\epsilon_3}$.
```

Comment:
you are probably better off if you write these formulas using `\sum` and with exponents `\alpha_{i,d}`, d=1,2,3. This way you don't have to use different greek symbols for each term, and you can identify exponents and coefficients in the documentation of the constructor.